package.json

```diff
@@ -48,6 +48,7 @@
     "@atlaskit/reduced-ui-pack": "^8.19.0",
     "@atlassian/aui": "^9.4.0",
     "@octokit/auth-app": "^2.6.0",
+    "@octokit/graphql-schema": "^10.73.0",
```

Comment:
This is for GraphQL Types, we are still getting rid of Octokit Client, but we need types, so we are going to reuse its GraphQL Types